src/logger.ts

```diff
@@ -98,17 +109,22 @@ export class FormatLogger implements LogSink {
     this._format = format;
   }
 
-  log(level: LogLevel, ...args: unknown[]): void {
-    console[level](...this._format(level, ...args));
+  log(level: LogLevel, context: Context | undefined, ...args: unknown[]): void {
+    console[level](
+      ...this._format(level, ...stringified(context).concat(args)),
+    );
   }
 }
 
 /**
  * A console logger that prefixes log lines with a log level.
  */
 export const nodeConsoleLogSink: LogSink = {
-  log(level: LogLevel, ...args: unknown[]): void {
-    console[level](logLevelPrefix[level], ...args.map(normalizeArgument));
+  log(level: LogLevel, context: Context | undefined, ...args: unknown[]): void {
+    console[level](
+      logLevelPrefix[level],
+      ...stringified(context).concat(args).map(normalizeArgument),
```

Comment:
I do not think you need to call `normalizeArgument` on the stringified context

Reply:
Good point, then I can unroll this one too.